spirv_msl.cpp

```diff
@@ -4871,7 +4984,28 @@ void CompilerMSL::emit_store_statement(uint32_t lhs_expression, uint32_t rhs_exp
 	bool transpose = lhs_e && lhs_e->need_transpose;
 
 	// No physical type remapping, and no packed type, so can just emit a store directly.
-	if (!lhs_remapped_type && !lhs_packed_type)
+	if (lhs_e!=nullptr && lhs_e->loaded_from==builtin_primitive_indices_id)
+	{
+		auto &execution = get_entry_point();
+		auto str = to_expression(lhs_expression);
+		str = str.substr(str.find_first_of('[')+1, str.find_last_of(']') - str.find_first_of('[') - 1);
```

Comment:
This code look quite bad, but I don't see better way, atm.
Need some way to access OpAccessChain parameters, or some way to emit shorten chain.

Reply:
Hey, small bump to this. 
My question basically: is there any way to reach original `OpAccessChain` with all arguments?
Needed here and for DX mesh-shader as well.
Thanks!